tools/staramr/staramr_search.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="staramr_search" name="staramr" version="@VERSION@">
+<tool id="staramr_search" name="staramr" version="@VERSION@+galaxy1">
```

Comment:
Is there a reason you updated the version like this? I thought for quick bug fixes in the wrapper we should leave the version alone (so anyone who installs the tool will get the latest version). But, I don't know if this has changed (I haven't made Galaxy tools in a while).

Reply:
They have changed to the **+galaxy#** format a while ago but was not 100% sure if this case needs to have it. 

What do you think?

Reply:
I don't think it matters too much. So let's just stick with what you've submitted now. Looks like it passed so I'm merging.

Reply:
Hi, the version should be increased as soon as the results could be impacted. If you fix help text and such a version increase is not needed,

Reply:
Thanks for the clarification!

Reply:
Yes. Thank you 👍